src/lib/_common.scss

```diff
@@ -117,34 +122,46 @@ $header-height: 48px;
     }
 }
 
-.bp {
-    align-items: center;
+.bp-body {
     background-color: $ffive;
-    border: 0 none;
     bottom: 0;
     display: flex;
-    flex-direction: column;
-    justify-content: center;
+    flex-direction: row;
     left: 0;
-    margin: 0;
-    outline: none;
-    -webkit-overflow-scrolling: touch;
-    padding: 0;
     position: absolute;
     right: 0;
     top: 0;
 
-    // Pseudo fullscreen for iOS Safari which doesn't support the fullscreen API
-    &.bp-fullscreen-unsupported {
-        bottom: 0;
-        left: 0;
-        position: fixed;
-        right: 0;
-        top: 0;
+    .bp-thumbnails-container {
+        border-right: solid 1px $seesee;
+        flex: 0 0 180px;
     }
 
-    &.bp-dark {
-        background-color: $black;
+    .bp {
+        align-items: center;
+        border: 0 none;
```

Comment:
Is there some other way to do this rather than refactoring the DOM and css for this? This will affect all viewers. Possibly by adding a class to .bp, and overriding the css accordingly